### PR TITLE
Adds DoR and DoD

### PR DESCRIPTION
**Pull request by SlnPons**

### CODE REVIEW

**Comment by Thykof** [comment]
For the DOD, here is what I suggest: add a pull request template instead inside the issue template (so that the checklist if validated Before merging)

```
DOD:
- [ ] QA review process is described
- [ ] CI is green
- [ ] code is tested
- [ ] front: stories are added
- [ ] PR is relatively small
- [ ] 2 approvals
- [ ] documentation is updated
```

**Comment by SlnPons** [comment]
> For the DOD, here is what I suggest: add a pull request template instead inside the issue template (so that the checklist if validated Before merging)
> 
> ```
> DOD:
> - [ ] QA review process is described
> - [ ] CI is green
> - [ ] code is tested
> - [ ] front: stories are added
> - [ ] PR is relatively small
> - [ ] 2 approvals
> - [ ] documentation is updated
> ```

Why not. But I'm not 100% convinced because: 
* by doing that developers won't take a look at the task before putting it in CR and will only check his code. However the vision and user flow are key and so must be read again and again to make sure we're going in the right direction. 
* by doing that we assume that devs are only responsible for this list you did. But it's not true, all the team is responsible for everything.
* I think I don't understand you're 4rd point: front-end stories are added?! --> once again I think it adds complexity to have DoD for back-end, for front-end and for features. One DoD for the team for this particular objective is simpler. 

WDYT?
Let's keep it simple and only keep it in one piece and in one place for now.

**Comment by SlnPons** [comment]
It seems that markdown is not working for the hyperlink text... any idea what I could do? 😅
If it's because of the colour, happy to simply remove it! 
If it's about the toggle part, ping me I'll revisit this so the issue is not that heavy...

File: .github/ISSUE_TEMPLATE/Task.md [hunk 1]
```diff
@@ -3,23 +3,71 @@ name: Task
...
 
 **QA testing**
+
 *Does this task require some QA tests ?*
 *If yes, explain how to validate it*
+
+<details>
+  <summary>$\textcolor{Red}{\textsf{DoD}}$ </summary>
+Before putting this ticket in code review, tick all the boxes bellow.
+  More details [here] (https://www.notion.so/massa-innoteam/Plan-for-the-mainnet-c574da44a4854eb3841a5f2e93a2977c?pvs=4#e7db6fa53fa84264954075011432ce70) & [here] (https://www.notion.so/massa-innoteam/Industrialization-of-Frontend-0f7425f14cd3490a949f31978916ee41?pvs=4) if needed
```
Comment (Thykof) [inline .github/ISSUE_TEMPLATE/Task.md:56]: you can try:

```suggestion
  More details here: https://www.notion.so/massa-innoteam/Plan-for-the-mainnet-c574da44a4854eb3841a5f2e93a2977c?pvs=4#e7db6fa53fa84264954075011432ce70 & here: https://www.notion.so/massa-innoteam/Industrialization-of-Frontend-0f7425f14cd3490a949f31978916ee41?pvs=4 if needed
```

or

```suggestion


  More details [here] (https://www.notion.so/massa-innoteam/Plan-for-the-mainnet-c574da44a4854eb3841a5f2e93a2977c?pvs=4#e7db6fa53fa84264954075011432ce70) & [here] (https://www.notion.so/massa-innoteam/Industrialization-of-Frontend-0f7425f14cd3490a949f31978916ee41?pvs=4) if needed
 
 
```

**Comment by mazmassa** [comment]
LGTM!

Just a comment about: `- [ ] PR is relatively small.`

I don't see that as an **action** it self and it's hard to say when a code is small or not. I would say that we could return a code saying: too much but it will remains a Dev good practice to send or not rather than a checklist action. Not sure I was clear...

**Review by mazmassa — APPROVED** [review]

**Pull request closed without merging**